src/engagement_db_to_coda/engagement_db_to_coda.py

```diff
@@ -140,11 +138,12 @@ def _update_engagement_db_message_from_coda_message(engagement_db, engagement_db
 
 
 @firestore.transactional
-def _sync_message_to_coda(transaction, engagement_db, coda, coda_config, message_id):
+def _sync_next_message_to_coda(transaction, engagement_db, coda, coda_config, last_seen_message):
     """
     Syncs a message from an engagement database to Coda.
 
     This method:
+     - Gets the least recently updated message that was last updated after `last_updated_since`.
```

Comment:
There isn't variable name `last_updated_since` although it makes sense. You can say  ...after the last update or sth similar

Reply:
great catch, this docstring is out of date so I've changed this to last_seen_message